W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/create-basic-integration.adoc

+
name:: _inbound-request_
doc:name:: _HTTP Config_

> . Press Return to move to a new line and `tab` to add an indentation.

Author: ```suggestion
. On a new line, press Tab to add an indentation.
```

--
Using *Run* > *Start without Debugging* causes an error.
--

> . Open the Terminal window Ctrl (Mac) followed by a backtick (`), and verify that the deployment was successful.

Author: ```suggestion
. Open the Terminal window Ctrl (Mac) followed by a backtick (`), and verify that the deployment succeeded.
```
